### PR TITLE
(feat) configure the pull robot to merge upstream politely

### PR DESCRIPTION
**Pull request by Floppy**
See docs at https://github.com/wei/pull#readme

### CODE REVIEW

File: .github/pull.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+version: "1"
+rules:
+  - base: master
+    upstream: alphagov:master
+    mergeMethod: merge
```
Comment (rjw1) [inline .github/pull.yml]: Could we have a new line :)

Comment (Floppy) [reply .github/pull.yml]: Oh go on then.

Comment (Floppy) [reply .github/pull.yml]: I assume that's where you mean...

Comment (rjw1) [reply .github/pull.yml]: no I meant at the end of the file

Comment (Floppy) [reply .github/pull.yml]: Ooh, I see. I did think it was an odd style. :)

Comment (Floppy) [reply .github/pull.yml]: done :)

**Pull request merged**